### PR TITLE
ignore model background in their bounding boxes

### PR DESCRIPTION
**Pull request by hanbollar**
## Linking

Fixes https://github.com/Volumetrics-io/mrjs/issues/434
Fixes https://github.com/Volumetrics-io/mrjs/issues/450
Related to https://github.com/Volumetrics-io/mrjs-landing/pull/1

## Problem

Now that models are part of mr-div instead of pure mr-entity their bounding boxes include the background plane default to mr-div.

we had moved models to mr-div for physics + better layout capabilities in mr-panel, but the use of entity.background isnt helpful from a physics perspective and isnt used in that regard

## Solution

ignore background specifically for mr-model when updating the model's bounding box

Before:
Screenshot 2024-03-14 at 11 43 14 AM

After:

Screenshot 2024-03-14 at 11 42 41 AM

## Note

This pr also includes the addition of a copy of mrjsio.html in the /sites/ sub folder of /samples/ --> in future i'll add a trigger so that auto updates based on the landing page's code so we can keep it in our testing cycle

------------

## Required to Merge

- [x] **PASS** - all necessary actions must pass (excluding the auto-skipped ones)
- [x] **TEST IN HEADSET** - [main dev-testing-example](https://github.com/Volumetrics-io/mrjs/tree/main/samples/index.html) and any of the other [examples](https://github.com/Volumetrics-io/mrjs/tree/main/samples/examples) still work as expected
- [x] **VIDEO** - if this pr changes something visually - post a video here of it in headset-MR and/or on desktop (depending on what it affects) for the reviewer to reference.
  - has a visual change - posting photo for better clarity
- [x] **TITLE** - make sure the pr's title is updated appropriately as it will be used to name the commit on merge
~~- [ ] **BREAKING CHANGE**~~
  - make a pr in the [documentation repo](https://github.com/Volumetrics-io/documentation) that updates the manual docs to match the breaking change
  - note: the docs underneath the `Javascript API` section come automated from the jsdocs inline with the code itself
  - link the pr of the documentation repo here: *#pr*
  - that documentation repo pr must be approved by `@lobau`

### CODE REVIEW

**Comment by hanbollar** [comment]
looks like the issue is specific to the rotating colliders and what is closest

since the models are rotating their colliders are coming in the way of that of the objects themself - gonna noodle thru this a bit in relation to if we can grab a specific link versus closest

**Comment by hanbollar** [comment]
also the model colliders are huge compared to their own size - still looking into why the diff in size expectation

**Comment by hanbollar** [comment]
additional noodle: part of it is due to resize changing scale of bbox though reload of page fixes it

File: src/core/componentSystems/PhysicsSystem.js [hunk 1]
```diff
@@ -49,36 +49,31 @@ export class PhysicsSystem extends MRSystem {
...
-
-        for (const entity of this.registry) {
-            if (entity.physics?.body == null) {
-                continue;
-            }
-            this.updateBody(entity);
-        }
-
-        this.updateDebugRenderer();
+        this.eventUpdate();
```
Comment (michaelthatsit) [inline src/core/componentSystems/PhysicsSystem.js:76]: oof nah we can't do it this way. `eventUpdate` is only meant to be called on events, not per frame.

**Review by michaelthatsit — CHANGES_REQUESTED** [review]
The changes you made in `eventUpdate` & `update` reintroduce performance bugs we resolved a couple weeks ago. Are they absolutely necessary for this fix to work?

File: src/core/componentSystems/PhysicsSystem.js [hunk 2]
```diff
@@ -49,36 +49,31 @@ export class PhysicsSystem extends MRSystem {
...
+
         for (const entity of this.registry) {
             if (entity.physics?.body == null) {
                 continue;
             }
-            if (entity instanceof MRModel) {
-                this.updateSimpleBody(entity);
-            } else if (entity instanceof MRDivEntity) {
-                this.updateUIBody(entity);
-            }
```
[the remark this answers is not in the document]
Comment (hanbollar) [reply src/core/componentSystems/PhysicsSystem.js:61]: oh these arent required - was trying to simplify the update but if the difference of separation is needed, can still do that 👍

**Review by michaelthatsit — APPROVED** [review]

**Pull request merged**